File: app/src/main/java/guillermo/com/pugme/callbacks/FetchPugsUrlsCallBack.java
package guillermo.com.pugme.callbacks;

import java.util.List;

/**
 * Created by guillermo.rosales on 21/01/16.
 */
public interface FetchPugsUrlsCallBack {

    void onPugsUrlsFetched(List<String> urls);

    void onReportsFetchFailed(String message);
}

File: app/src/main/java/guillermo/com/pugme/callbacks/FetchPugsUrlsCallBack.kt
package guillermo.com.pugme.callbacks

import java.util.*

/**
 * Created by guillermo.rosales on 21/01/16.
 */
interface FetchPugsUrlsCallBack {

    fun onPugsUrlsFetched(urls: ArrayList<String>?)

    fun onReportsFetchFailed(message: String)
}
